src/web3mail/sendEmail.ts

```diff
@@ -128,9 +136,13 @@ export const sendEmail = async ({
     // Push requester secrets
     const emailSubjectId = generateSecureUniqueId(16);
     const emailContentId = generateSecureUniqueId(16);
+    const contentTypeId = generateSecureUniqueId(16);
     await iexec.secrets.pushRequesterSecret(emailSubjectId, vEmailSubject);
     await iexec.secrets.pushRequesterSecret(emailContentId, vEmailContent);
-
+    await iexec.secrets.pushRequesterSecret(
+      contentTypeId,
+      `{"contentType": "${vContentType}"}`
```

Comment:
use JSON stringify to ensure compact notation and leave room to add other options
```suggestion
      JSON.stringify({contentType: vContentType})
```